src/libcollections/vec_map.rs

```diff
@@ -13,6 +13,8 @@
 
 #![allow(missing_docs)]
 
+pub use self::Entry::*;
```

Comment:
Hmm, I basically was copy-pasting from BTreeMap, I didn't even notice the `pub`.

I'm going to refactor this all in my next PR anyway.